fleet_vehicle_category/tests/test_vehicle_category.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import TransactionCase
+
+
+class TestFleetVehicleCategory(TransactionCase):
+    def setUp(self):
```

Comment:
Hey, these added unit tests don't really add much value.
There's no business logic in this module, just fields. We don't need to test the ORM